### PR TITLE
Add arrow on menu instead of going to page everytime you click

### PR DESCRIPTION
**Pull request by NeOMakinG**
Fix https://github.com/PrestaShop/docs/issues/584

### CODE REVIEW

File: src/themes/hugo-theme-learn/_src/js/theme.js [hunk 1]
```diff
@@ -1 +1,29 @@
...
+const menuItems = document.querySelectorAll('ul.topics li.isParent > a');
+
+function toggleIcon(icon) {
+  if(icon.classList.contains('fa-angle-up')) {
+    icon.classList.remove('fa-angle-up')
+    icon.classList.add('fa-angle-down')
+  }else {
+    icon.classList.add('fa-angle-up')
+    icon.classList.remove('fa-angle-down')
+  }
```
Comment (Progi1984) [inline src/themes/hugo-theme-learn/_src/js/theme.js:12]:
```suggestion
  if (icon.classList.contains('fa-angle-up')) {
    icon.classList.remove('fa-angle-up')
    icon.classList.add('fa-angle-down')
  } else {
    icon.classList.add('fa-angle-up')
    icon.classList.remove('fa-angle-down')
  }
```

File: src/themes/hugo-theme-learn/_src/sass/components/_sidebar.scss [hunk 2]
```diff
@@ -142,13 +142,29 @@ $text-trailing-indent: .5rem;
           background: $active-block-bg;
         }
 
+
+
```
Comment (Progi1984) [inline src/themes/hugo-theme-learn/_src/sass/components/_sidebar.scss:146]:
```suggestion
```

Comment (Progi1984) [inline src/themes/hugo-theme-learn/_src/sass/components/_sidebar.scss:146]:
```suggestion

```

File: src/themes/hugo-theme-learn/_src/sass/components/_sidebar.scss [hunk 3]
```diff
@@ -142,13 +142,29 @@ $text-trailing-indent: .5rem;
...
         > ul > li {
```
Comment (Progi1984) [inline src/themes/hugo-theme-learn/_src/sass/components/_sidebar.scss:147]:
```suggestion
        > ul > li {
```

**Comment by NeOMakinG** [comment]
I seen a bug, need to fix it

**Comment by matks** [comment]
I checked out the branch to see how it works.

First of all: nice job @NeOMakinG 👍 

I have a series of questions/suggestions but I dont know if they are mandatory or nice-to-have. Maybe @eternoendless should answer ?

### Dropdown on top categories

When I open the devdocs homepage I see all the arrows. It feels a bit strange to have these dropdowns enabled on top categories. In subcategories, yeah, but on top categories it feels weird to me.

Capture d’écran 2020-06-24 à 14 08 43

### No animation

When clicking on a arrow, the submenu appears instantly (there is no "opening" animation). Since categories are not very well separated in a visual way, if you do not pay close attention, sometimes you dont know exactly what happened. I'm sorry it's not clear how I explain it.

Having a "opening" animation would make it easier for the eyes to see what was opened.

### Dropdown does not close on click

If I open a subcategory, it closes the other subcategories of the same level => perfect

However sometimes I want to close a subcategory, because it's a big one and I want to have a look at all subcategories available, but clicking on the "close" / "top" arrow does not close the subcategory. So it's a "open-only" dropdown 😄

**Comment by NeOMakinG** [comment]
I'll take a look for the dropdown not closing issue, thanks for your feedback @matks 

Btw, the close of subcategories of the same level isn't wanted, I need to find why 😄

**Comment by matks** [comment]
> Btw, the close of subcategories of the same level isn't wanted, I need to find why 😄

Well, it makes sense. There are too many things on the left menu, you cannot keep 2 subcategories of same level opened, else you have to scroll to see it all.

So I suggest to keep it 😉

**Comment by matks** [comment]
Fixed by building assets
```
$ src/themes/hugo-theme-learn/_src
$ npm i
$ npm run build
```

**Comment by NeOMakinG** [comment]
> Fixed by building assets
> 
> ```
> $ src/themes/hugo-theme-learn/_src
> $ npm i
> $ npm run build
> ```

I'm searching which asset I need to push

**Comment by matks** [comment]
@NeOMakinG Ready for new QA ? I will be your QA 😛

**Comment by NeOMakinG** [comment]
@matks yes do it!

**Comment by matks** [comment]
I validate ✅

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**